### PR TITLE
Column data model (5)

### PR DESCRIPTION
**Pull request by jacobp100**

### CODE REVIEW

File: packages/ag-charts-enterprise/src/series/radial-bar/radialBarSeries.ts [hunk 1]
```diff
@@ -175,6 +175,10 @@ export class RadialBarSeries extends _ModuleSupport.PolarSeries<
                 ),
                 ...extraProps,
             ],
+            formatIntoColumns: false,
+            doNotFormatIntoRows: false,
+            // formatIntoColumns: true,
```
Comment (alantreadway) [inline packages/ag-charts-enterprise/src/series/radial-bar/radialBarSeries.ts:180]: Nit: ?

File: packages/ag-charts-enterprise/src/series/radial-bar/radialBarSeries.ts [hunk 2]
```diff
@@ -225,8 +229,12 @@ export class RadialBarSeries extends _ModuleSupport.PolarSeries<
 
         const angleStartIndex = dataModel.resolveProcessedDataIndexById(this, `angleValue-start`);
         const angleEndIndex = dataModel.resolveProcessedDataIndexById(this, `angleValue-end`);
-        const angleRangeIndex = dataModel.resolveProcessedDataIndexById(this, `angleValue-range`);
         const angleRawIndex = dataModel.resolveProcessedDataIndexById(this, `angleValue-raw`);
+        // const angleStartValues = dataModel.resolveColumnById(this, `angleValue-start`, processedData);
```
Comment (alantreadway) [inline packages/ag-charts-enterprise/src/series/radial-bar/radialBarSeries.ts:233]: Nit: ?

File: packages/ag-charts-enterprise/src/series/radial-bar/radialBarSeries.ts [hunk 3]
```diff
@@ -273,14 +281,22 @@ export class RadialBarSeries extends _ModuleSupport.PolarSeries<
...
-            const { datum, keys, values, aggValues } = group;
-
+        const { rawData } = processedData;
+        processedData.data.forEach((group) => {
+            // const { aggValues } = group;
+            const { keys, values, aggValues } = group;
+            const datumIndex = group.index as number;
+            const datum = rawData[datumIndex];
+
+            // const radiusDatum = keys![datumIndex][0];
```
Comment (alantreadway) [inline packages/ag-charts-enterprise/src/series/radial-bar/radialBarSeries.ts:291]: Nit: ?

**Review by alantreadway — APPROVED** [review]
LGTM

File: packages/ag-charts-enterprise/src/series/radial-bar/radialBarSeries.ts (hunk 1 above)
Comment (jacobp100) [reply packages/ag-charts-enterprise/src/series/radial-bar/radialBarSeries.ts:180]: I got this half working but need to do more fixes - so I just left this in to make it easier to re-enable. Will be fixed in the next PR or so

**Pull request closed without merging**